internal/devices/pnp.go

```diff
@@ -61,3 +66,72 @@ func execPnPInstallDriver(ctx context.Context, vm *uvm.UtilityVM, driverDir stri
 	log.G(ctx).WithField("added drivers", driverDir).Debug("installed drivers")
 	return nil
 }
+
+func execModprobeInstallDriver(ctx context.Context, vm *uvm.UtilityVM, driverDir string) error {
```

Comment:
Move LCOW install support routines out of pnp.go

Reply:
Done